source/utils/validatePrivateKey.ts

```diff
@@ -0,0 +1,25 @@
+import { ethers } from 'ethers';
+
+export const validatePrivateKeyValue = (privKey: string) => {
```

Comment:
this can be "simplified" with something like 

```typescript
export const validatePrivateKeyValue = (privKey: string) => {
  const checkAddress = privKey.slice(0, 2);
  const adjustedPrivKey = checkAddress === '0x' ? `0x${privKey}` : privKey;

  try {
    new ethers.Wallet(testKey);
  } catch (error) {
    return false;
  }

  return true;
};
```